activejob/lib/active_job/railtie.rb

```diff
@@ -36,6 +36,12 @@ class Railtie < Rails::Railtie # :nodoc:
       end
     end
 
+    initializer "active_job.set_filter_arguments" do
+      ActiveSupport.on_load(:active_job) do
+        self.filter_arguments += Rails.application.config.filter_parameters
```

Comment:
This is a bit different than "Defaults to `Rails.application.config.filter_parameters`."  This always appends `Rails.application.config.filter_parameters` to `filter_arguments`.  This might be what we want after all, but if not, you can do something like:
```ruby
self.filter_arguments = Rails.application.config.filter_parameters if filter_arguments.empty?
```